runtime/gc_base/FinalizerSupport.cpp

```diff
@@ -869,6 +887,23 @@ int j9gc_finalizer_startup(J9JavaVM * vm)
 }
 
 /**
+ * check if processing reference is active
```

Comment:
Please document parameters and the return value.

Reply:
Nitpick: `check` -> `Check`